JustSaying.IntegrationTests/WhenRegisteringASqsSubscriber/MessageLockStore.cs

```diff
@@ -17,13 +18,18 @@ public MessageLockResponse TryAquireLockPermanently(string key)
                 _store.Add(key, 1);
             }
 
-            return new MessageLockResponse { DoIHaveExclusiveLock = canAquire };
+            var response = new MessageLockResponse { DoIHaveExclusiveLock = canAquire };
```

Comment:
Shouldn't this be setting the expiry properties? And why have we got two methods that do that same thing but are named differently? And if we are going to set expiry properties, then we should implement the expiry!

Reply:
No idea. I've just moved this code to conform to the interface change. And it's a test object.

Reply:
Ah, didn't see that it's a test fake. Nevermind then.